x-pack/solutions/observability/plugins/observability_ai_assistant_app/server/functions/query/validate_esql_query.ts

```diff
@@ -47,30 +49,22 @@ export async function runAndValidateEsqlQuery({
     return 'text' in error ? error.text : error.message;
   });
 
-  return client.transport
-    .request({
-      method: 'POST',
-      path: '_query',
-      body: {
-        query,
-      },
-    })
-    .then((res) => {
-      const esqlResponse = res as ESQLSearchResponse;
+  try {
+    const res = await client.esql.query({ query, drop_null_columns: true }, { signal });
```

Comment:
@dgieselaar I assume we the ES client didn't have `esql.query` previously which is why we used the `transport` api. LMK if there's any reason we should keep using this.